en-US/firefox-ios.xliff

```diff
@@ -3951,6 +3946,23 @@ Private Tab</target>
       </trans-unit>
     </body>
   </file>
+  <file original="Shared/Supporting Files/en.lproj/CustomizeFirefoxHome.strings" datatype="plaintext" source-language="en" target-language="en-US">
+    <header>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="14.0.1" build-num="14A400"/>
+    </header>
+    <body>
+      <trans-unit id="Settings.Home.Option.ThoughtProvokingStories.subtitle.v116" xml:space="preserve">
```

Comment:
@thatswinnie: Pocket is brand name and like other brand names, should be replaced by a placeholder to avoid translation.

Reply:
@Delphine the string is fixed now on main

Reply:
Thank you!